ARDemos/ARDemos/BottomBar.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class BottomBar: UIView {
+
+    private let scrollView = UIScrollView()
+    private var models: [Model]?
+    
+    var onTap: ((String) -> Void)?
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setUpViews()
+        setUpLayout()
+    }
+
+    private func setUpViews() {
+        addSubview(scrollView)
+    }
+
+    private func setUpLayout() {
+        applyConstraint(for: .top)
+        applyConstraint(for: .right)
+        applyConstraint(for: .bottom)
+        applyConstraint(for: .left)
+    }
+    
+    private func applyConstraint(for attribute: NSLayoutAttribute) {
```

Comment:
you can use the name of this method to increase the readability of this code. 
If you use `func constraintToSuperview(_ attribute: NSLayoutAttribute)` the code above would read better: 

```
private func setUpLayout() {
    constraintToSuperview(.top)
    constraintToSuperview(.right)
    constraintToSuperview(.bottom)
    constraintToSuperview(.left)
}
```

Reply:
:+1:  181aa0a73f4bd1b497c995f56931a3aed0a4c497